### ISSUE TITLE
最新的整合卡在这个阶段不会动了

### ISSUE THREAD

**Issue by XqnoNO4** [YYuX-1145/Bert-VITS2-Integration-package#8]
F:\Bert-VITS2-Integration-Package>%PYTHON% train_ms.py -c ./configs\config.json
INFO:OUTPUT_MODEL:{'train': {'log_interval': 10, 'eval_interval': 100, 'seed': 52, 'epochs': 1000, 'learning_rate': 0.00
02, 'betas': [0.8, 0.99], 'eps': 1e-09, 'batch_size': 8, 'fp16_run': False, 'lr_decay': 0.999875, 'segment_size': 16384,
 'init_lr_ratio': 1, 'warmup_epochs': 0, 'c_mel': 45, 'c_kl': 1.0}, 'data': {'use_mel_posterior_encoder': False, 'traini
ng_files': 'filelists/train.list', 'validation_files': 'filelists/val.list', 'max_wav_value': 32768.0, 'sampling_rate':
44100, 'filter_length': 2048, 'hop_length': 512, 'win_length': 2048, 'n_mel_channels': 128, 'mel_fmin': 0.0, 'mel_fmax':
 None, 'add_blank': True, 'n_speakers': 1, 'cleaned_text': True, 'spk2id': {'MuelsyseZH': 0}}, 'model': {'use_spk_condit
ioned_encoder': True, 'use_noise_scaled_mas': True, 'use_mel_posterior_encoder': False, 'use_duration_discriminator': Tr
ue, 'inter_channels': 192, 'hidden_channels': 192, 'filter_channels': 768, 'n_heads': 2, 'n_layers': 6, 'kernel_size': 3
, 'p_dropout': 0.1, 'resblock': '1', 'resblock_kernel_sizes': [3, 7, 11], 'resblock_dilation_sizes': [[1, 3, 5], [1, 3,
5], [1, 3, 5]], 'upsample_rates': [8, 8, 2, 2, 2], 'upsample_initial_channel': 512, 'upsample_kernel_sizes': [16, 16, 8,
 2, 2], 'n_layers_q': 3, 'use_spectral_norm': False, 'gin_channels': 256}, 'model_dir': './logs\\./OUTPUT_MODEL', 'cont'
: False}
WARNING:OUTPUT_MODEL:F:\Bert-VITS2-Integration-Package is not a git repository, therefore hash value comparison will be
ignored.
一直卡在这个阶段是什么问题啊
